fastlane/Fastfile

```diff
@@ -10,3 +10,9 @@ lane :rubocop do
   sh("cd ..;bundle install")
   sh("cd ..;bundle exec rubocop")
 end
+
+lane :tslint do
+  desc "TSLint the whole project to ensure no style violations"
+  sh("cd ..;npm install")
```

Comment:
how do we guarantee that `npm` is installed?

Reply:
good point. I following bundle as an example, but i guess there's a higher assumption that it's already installed if you can run fastlane.

Any suggestions? I'll look into it in the meantime.

Reply:
@taquitos wouldn't the command fail, thus failing the fastlane execution? If so, that should be OK. doing a `brew install npm` might be overkill.

Reply:
Yeah, this is fine.
Let's add to the rake file whatever is the standard way of getting npm

Reply:
done